romancal/datamodels/schemas/image_files/image.schema.yaml

```diff
@@ -0,0 +1,327 @@
+            continuation_id:
+              title: Continuation of previous program
+              type: integer
+        observation:
+          title: Observation identifiers
+          type: object
+          properties:
+            date:
+              title: "[yyyy-mm-dd] UTC date at start of exposure"
+              type: string
+            time:
+              title: "[hh:mm:ss.sss] UTC time at start of exposure"
+              type: string
+            date_beg:
+              type: string
+              title: "Date-time start of exposure"
+              # Despite what the title might lead you to believe, this keyword
+              # actually contains a full datetime string.
+            date_end:
+              type: string
+              title: "Date-time end of exposure"
+              # Despite what the title might lead you to believe, this keyword
+              # actually contains a full datetime string.
+            obs_id:
+              title: Programmatic observation identifier
+              type: string
+            visit_id:
+              title: Visit identifier
+              type: string
+            program_number:
+              title: Program number
+              type: string
+            observation_number:
+              title: Observation number
+              type: string
+            visit_number:
+              title: Visit number
+              type: string
+            visit_group:
+              title: Visit group identifier
+              type: string
+            sequence_id:
+              title: Parallel sequence identifier
+              type: string
+            activity_id:
+              title: Activity identifier
+              type: string
+            exposure_number:
+              title: Exposure request number
+              type: string
+            bkgdtarg:
+              title: Background target
+              type: boolean
+            template:
+              title: Observation template used
+              type: string
+            observation_label:
+              title: Proposer label for the observation
+              type: string
+            observation_folder:
+              title: Name of the APT observation folder
+              type: string
+        visit:
+          title: Visit information
+          type: object
+          properties:
+            engineering_quality:
+              title: Engineering data quality indicator from EngDB
+              type: string
+              enum: [OK, SUSPECT]
+            type:
+              title: Visit type
+              type: string
+              enum: [GENERIC, PARALLEL_DARK_CALIBRATION, PARALLEL_EXTERNAL_CALIBRATION,
+                     PARALLEL_PURE, PARALLEL_SLEW_CALIBRATION, PRIME_STATION_KEEPING,
+                     PRIME_TARGETED_FIXED, PRIME_TARGETED_MOVING, PRIME_UNTARGETED,
+                     PRIME_WFSC_ROUTINE, PRIME_WFSC_SENSING_CONTROL,
+                     PRIME_WFSC_SENSING_ONLY, PRIOR_MOMENTUM_UNLOAD]
```

Comment:
Were these values provided by the INS team?

Reply:
No, these are the old JWST values left as a guide for future tweaking. I can remove them, if desired.

Reply:
I removed this enum -  we can create a new one when new values are determined.